internal/provider/resource_build_test.go

```diff
@@ -224,7 +224,7 @@ resource "apko_build" "foo" {
 				resource.TestCheckResourceAttr("apko_build.foo", "repo", repostr),
 				resource.TestCheckResourceAttr("apko_build.foo", "image_ref",
 					// With pinned packages we should always get this digest.
-					repo.Digest("sha256:3cf315a06325088c5a7700de3d47b6968ce0b7d989e880f5eac42e9eb839b147").String()),
+					repo.Digest("sha256:73959382142e5648feb87ccaa0acc69628fa9b650927642a1583af71eb9e29ae").String()),
```

Comment:
these change due to the different gzip implementation picked up with the latest `go-apk` and `apko` revisions.

Reply:
(according to @jonjohnsonjr)